nemo/collections/asr/parts/manifest.py

```diff
@@ -74,10 +76,33 @@ def item_iter(
 
                 yield item
 
+def __parse_open_stt(line: str, manifest_file: str) -> Dict[str, Any]:
+    manifest_path, _ = split(manifest_file)
+    line_list = line.split(",")
+    assert len(line_list)==3, f"Incorrect fields number: {len(line_list)}"
+    item = dict(
+        offset=None,
+        speaker=None,
+        orig_sr=None,
+    )
+
+    item['audio_file'] = line_list[0]
+    if item['audio_file'][0] != "/":
```

Comment:
Forward slash is not always the path seperator, use os.sep everywhere

Reply:
fixed by isabs()